.gitignore

```diff
@@ -9,7 +9,7 @@ zos.dev-*.json
 
 # Artifacts
 unlock-app/src/artifacts/
-smart-contracts/build/contracts/*.json
+smart-contracts/build/
```

Comment:
I'm not quite sure what's happening here?

Reply:
truffle-security creates another build folder.
Instead of adding both, I trimmed what we had.  We don't want any build artifacts to be included.